ext/pdo/tests/pdo_014.phpt

```diff
@@ -3,7 +3,6 @@ PDO Common: PDOStatement SPL iterator
 --SKIPIF--
 <?php # vim:ft=php
 if (!extension_loaded('pdo')) die('skip');
-if (!extension_loaded('SPL')) die('skip SPL not available');
 if (!class_exists('IteratorIterator', false)) die('skip IteratorIterator class not present');
```

Comment:
The check for `IteratorIterator` can be dropped as well. This iterator is part of SPL, so it is always available.

Reply:
Sure. I'll search for similars.